uuid-kinds/src/lib.rs

```diff
@@ -45,6 +45,10 @@ macro_rules! impl_typed_uuid_kind {
 // Please keep this list in alphabetical order.
 
 impl_typed_uuid_kind! {
+    DownstairsRegionKind => "downstairs_region",
```

Comment:
This won't apply to any existing UUIDs for regions, right?  Only the new stuff you are
adding here?

Reply:
Correct, that's still work that is required.